### PR TITLE
[DOCFIX] Update cn version of Upgrade doc

### PR DESCRIPTION
**Pull request by voddle**
What changes are proposed in this pull request?
Update cn version of Upgrade doc.

Why are the changes needed?
There is no corresponding Chinese documentation for upgrade.

Does this PR introduce any user facing changes?
More Chinese users can access Alluxio documentation more easily.

### CODE REVIEW

**Comment by LuQQiu** [comment]
@voddle thanks for updating the doc

File: docs/cn/administration/Upgrade.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+group: Administration
+priority: 7
+---
+
+* Table of Contents
+{:toc}
+
+## 基础升级流程
+
+正常情况下，用户可以直接关闭当前的 Alluxio 进程，将 Alluxio 二进制文件更改为更新的版本，同时还按之前的方式配置 Alluxio 集群，并使用现有的日志文件夹/地址来启动 Alluxio 进程
```
Comment (jiacheliu3) [inline docs/cn/administration/Upgrade.md:14]:
```suggestion
正常情况下，用户可以直接关闭当前的 Alluxio 进程，将 Alluxio 二进制文件更改为更高的版本，同时还按之前的方式配置 Alluxio 集群，并使用现有的日志文件夹/地址来启动 Alluxio 进程
```

File: docs/cn/administration/Upgrade.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+进行升级。Alluxio 可读取以前的日志文件并自动恢复 Alluxio 元数据。
```
Comment (jiacheliu3) [inline docs/cn/administration/Upgrade.md:15]:
```suggestion
进行升级。Alluxio 可读取以前的日志文件并从 Journal 日志中自动恢复 Alluxio 元数据。
```

File: docs/cn/administration/Upgrade.md [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+
+以下两种情况下master日志无法向后兼容，需要采取额外的步骤来升级 Alluxio 集群：
+
+- 从 Alluxio 1.x 版本升级到 Alluxio 2.x 版本
+- 使用[内嵌日志]({{ '/cn/operation/Journal.html' | relativize_url}})的情况下从 Alluxio 2.3.x 及以下版本升级到 Alluxio 2.4.0 及以上版本
```
Comment (jiacheliu3) [inline docs/cn/administration/Upgrade.md:20]: 改成link到Journal.html#xxx embedded journal?

File: docs/cn/administration/Upgrade.md [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+
+本文档介绍了如何将 Alluxio 升级到非向后兼容版本。 即使是要升级到可向后兼容的版本，仍然建议按照以下步骤在升级前创建备份。
+
+## 创建当前版本的备份
+
+Alluxio-1.8.1 版本引入了日志备份功能。
```
Comment (jiacheliu3) [inline docs/cn/administration/Upgrade.md:26]:
```suggestion
Alluxio-1.8.1 版本引入了日志备份（backup）功能。
```

File: docs/cn/administration/Upgrade.md [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+注意，请不要在备份前修改 Alluxio 二进制文件。
+在运行 1.8.1 之前版本的 master 时，通过运行以下命令创建日志备份：
```
Comment (jiacheliu3) [inline docs/cn/administration/Upgrade.md:28]: 是不是只说了<=1.8.1版本时候怎么backup？那我觉得这里有点问题。用2.x版本的时候指令应该是一样的，你可以快速在自己电脑上验证一下，然后直接把“在运行 1.8.1 之前版本的 master 时”去掉

File: docs/cn/administration/Upgrade.md [hunk 6]
```diff
@@ -0,0 +1,101 @@
...
+```
+
+注意这里的 `${BACKUP_PATH}` 应该是类似 HDFS 地址的完整路径，可以被所有 Alluxio master 访问。
+如果要备份到本地文件系统路径，需将备份文件复制到所有 master 节点上的同一位置，然后通过本地备份文件路径启动所有 master。
+
+## 升级客户端和服务器
+
+Alluxio 2.x 版本对 RPC 层进行了重大修改，
+因此 2.0.0 之前版本的客户端不能与 2.0.0 之后版本的服务器一起运行，反之亦然。
+如果要使用 Alluxio-2.x 客户端需升级所有应用程序 。
```
Comment (jiacheliu3) [inline docs/cn/administration/Upgrade.md:65]:
```suggestion
如果要使用 Alluxio-2.x 客户端需升级所有应用程序 中的Alluxio客户端。
```

File: docs/cn/administration/Upgrade.md [hunk 7]
```diff
@@ -0,0 +1,101 @@
...
+1. 备份 Alluxio 中文件的元数据。请参阅有关 `backup` 命令的[文档]({{ '/cn/operation/Admin-CLI.html' | relativize_url }}#backup)。
+2. 停止 Alluxio 集群。
+```console
+$ ./bin/alluxio-stop.sh all
+```
+3. 更新所有应用程序的 Alluxio 客户端 jar 路径。例如`Yarn`, `Spark`, `Hive` 和 `Presto`，在 Cloudera Manager 的 "YARN (包括MR2)" 部分，在 "Configuration" 选项卡中，搜索参数 "Gateway Client Environment Advanced Configuration Snippet (Safety Valve) for hadoop-env.sh"。然后将以下行添加到脚本中：
+```console
+$ export HADOOP_CLASSPATH={{site.ALLUXIO_CLIENT_JAR_PATH}}:${HADOOP_CLASSPATH}
+```
+	如下所示：
```
Comment (jiacheliu3) [inline docs/cn/administration/Upgrade.md:77]: 缩进

File: docs/cn/administration/Upgrade.md [hunk 8]
```diff
@@ -0,0 +1,101 @@
...
+
+## 升级并从备份启动
+
+停止现有的 Alluxio 集群后，下载并解压新版本的 Alluxio 。
+从 `/conf` 目录拷贝旧的配置文件。然后通过以下命令将集群格式化：
+
+```console
+$ ./bin/alluxio format
+```
+- **警告：** 该操作会对 Alluxio worker 上的内存虚拟硬盘进行格式化（即：删除其中的内容）。
```
Comment (jiacheliu3) [inline docs/cn/administration/Upgrade.md:47]:
```suggestion
- **警告：** 该操作会对 Alluxio worker 上的内存虚拟硬盘（ramdisk）进行格式化（即：删除其中的内容）。
```

**Review by jiacheliu3 — CHANGES_REQUESTED** [review]
Added some minor comments, thanks for the work!

File: docs/cn/administration/Upgrade.md (hunk 3 above)
Comment (voddle) [reply docs/cn/administration/Upgrade.md]: 中文文档好像还没有对应更新

File: docs/cn/administration/Upgrade.md [hunk 9]
```diff
@@ -0,0 +1,101 @@
...
+
+## 基础升级流程
+
+正常情况下，用户可以直接关闭当前的 Alluxio 进程，将 Alluxio 二进制文件更改为更高的版本，同时还按之前的方式配置 Alluxio 集群，并使用现有的日志文件夹/地址来启动 Alluxio 进程
+进行升级。Alluxio 可读取以前的日志文件并从 Journal 日志中自动恢复 Alluxio 元数据。
+
+以下两种情况下master日志无法向后兼容，需要采取额外的步骤来升级 Alluxio 集群：
+
+- 从 Alluxio 1.x 版本升级到 Alluxio 2.x 版本
+- 使用[内嵌日志]({{ '/cn/operation/Journal.html#Configuring Embedded Journal' | relativize_url}})的情况下从 Alluxio 2.3.x 及以下版本升级到 Alluxio 2.4.0 及以上版本
```
Comment (jiacheliu3) [inline docs/cn/administration/Upgrade.md:20]: double check the URL, is it correct?

**Review by jiacheliu3 — APPROVED** [review]
LGTM, thanks!

**Comment by jiacheliu3** [comment]
alluxio-bot, merge this please

**Pull request merged**